README.md

```diff
@@ -0,0 +1,27 @@
+This repo is home for the SPA (Single Page Application) project templates which ship as part of ASP.NET Core.
+At the moment, those are `Angular` and `React` templates.
+
+Besides these two templates, we also want this repo to serve as a knowledge-base for all community-owned SPA project templates out there.
+We expect to maintain that list in the `spaProjectTempltes.md` file. If you know of any such project templates, which are popular, send us a PR with added information in that file, and we'll include it, it it meets the popularity bar.
+
+We try to keep the list of community owned SPA project templates up to date as well as want to make sure it is a good quality list for everyone to refer to.
+To keep the quality high, we have some requirements each proposed project template should comply to:
+- The project templates hould be available as a NuGet package
```

Comment:
Instead of trying to specify the exact criteria, maybe we should just say that submissions will be evaluated on a case-by-case basis to avoid adding templates that aren't of high-quality and widely enough used.